src/ff7_opengl.cpp

```diff
@@ -77,6 +77,12 @@ void ff7_init_hooks(struct game_obj *_game_object)
 	// Load Models atoi function
 	replace_call_function(ff7_externals.field_load_models_atoi, ff7_field_load_models_atoi);
 
+  // Restore Steam release behavior on character name screen when using gamepads in Steam Input mode
+  // Aali driver used to patch out these three functions to fix this issue
+  replace_function(ff7_externals.set_default_input_settings_save, noop);
+  replace_function(ff7_externals.keyboard_name_input, noop);
+  replace_function(ff7_externals.restore_input_settings, noop);
```

Comment:
Having those here will impact 1998 release as well. Better to move it where we do check if the game is the steam edition.

Reply:
@julianxhokaxhiu fixed.